openedx/features/idea/helpers.py

```diff
@@ -12,3 +18,41 @@ def upload_to_path(instance, filename, folder):
         filename=filename,
         folder=folder
     )
+
+
+def validate_image_dimension(image):
+    """
+    Validate image dimensions, raise validation error if anyone is different.
+    :param image: image which is being uploaded
+    :return: raise validation error otherwise None
+    """
+    image_width = getattr(image, 'width', None)
+    image_height = getattr(image, 'height', None)
+
+    errors = []
+
+    if image_width and image_width != IDEA_IMAGE_WIDTH:
+        errors.append('Width must be {px} px'.format(px=IDEA_IMAGE_WIDTH))
+
+    if image_height and image_height != IDEA_IMAGE_HEIGHT:
+        errors.append('Height must be {px} px'.format(px=IDEA_IMAGE_HEIGHT))
+
+    if errors:
+        raise ValidationError(', '.join(errors))
+
+
+def validate_file_size(file, max_allowed_size=IDEA_FILE_MAX_SIZE):
+    """
+    Validate maximum allowed file upload size, raise validation error if file size exceeds.
+    :param max_allowed_size: maximum allowed file size in bytes
+    """
+    size = getattr(file, 'size', None)
+    if size and max_allowed_size and max_allowed_size < size:
+        raise ValidationError(
+            _('File size must not exceed {size} MB').format(size=max_allowed_size / 1000 / 1000)
+        )
```

Comment:
Let's please make these methods generic and move these methods to common place
common/djangoapps/util/philu_utils.py

Reply:
Moved only `validate_file_size` to utils the other method has specific checks related to idea image dimensions, so leaving it.

Reply:
That's why I asked you to make these method generic by passing allowed width and allowed height, just like you passed max_allowed_size in validate_file_size

Reply:
This practice has never been followed before, making methods generic, consume extra effort, which is not estimated in the story. Let's discuss this in the next session of the code review meeting, so that everyone is on same page.

Reply:
We are following this approach in-fact a very basic thing that should be followed while writing code to make things generic so that it can be reused. The thing I am suggesting doesn't required too much effort but have more benefits.